docs/_static/theme_overrides.css

```diff
@@ -0,0 +1,13 @@
+/* override table width restrictions */
```

Comment:
Can we add a link to the source as a comment in here as well? I like that it's in the commit too.